index.html

```diff
@@ -51,6 +51,123 @@
+                    <div class="work-info-separator"></div>
+                    <div class="work-info-others">Back End Dev</div>
+                    <div class="work-info-separator"></div>
+                    <div class="work-info-others">2015</div>
+                </div>
+                <div class="work-description">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </div>
+                <ul class="work-stacks">
+                    <li class="stack"><a href="#" >HTML</a></li>
+                    <li class="stack"><a href="#" >Css</a></li>
+                    <li class="stack"><a href="#" >Javascript</a></li>
+                </ul>
+                <button class="work-link">See project</button>
+             </div>
+             <div class="work">
+                <img alt="work one" class="work-image" src="./assets/images/work2.svg"/>
+                <h2 class="work-title">Multi-Post <br> Stories</h2>
+                <div class="work-info">
+                    <div class="work-info-title">Canopy</div>
+                    <div class="work-info-separator"></div>
+                    <div class="work-info-others">Back End Dev</div>
+                    <div class="work-info-separator"></div>
+                    <div class="work-info-others">2015</div>
+                </div>
+                <div class="work-description">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </div>
+                <ul class="work-stacks">
+                    <li class="stack"><a href="#" >HTML</a></li>
+                    <li class="stack"><a href="#" >Css</a></li>
+                    <li class="stack"><a href="#" >Javascript</a></li>
+                </ul>
+                <button class="work-link">See project</button>
+                
+             </div>
+             <div class="work">
+                <img alt="work one" class="work-image" src="./assets/images/works.svg"/>
+                <h2 class="work-title">Tonic</h2>
+                <div class="work-info">
+                    <div class="work-info-title">Canopy</div>
+                    <div class="work-info-separator"></div>
+                    <div class="work-info-others">Back End Dev</div>
+                    <div class="work-info-separator"></div>
+                    <div class="work-info-others">2015</div>
+                </div>
+                <div class="work-description">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </div>
+                <ul class="work-stacks">
+                    <li class="stack"><a href="#" >HTML</a></li>
+                    <li class="stack"><a href="#" >Css</a></li>
+                    <li class="stack"><a href="#" >Javascript</a></li>
+                </ul>
+                <button class="work-link">See project</button>
```

Comment:
- [ ] Kindly follow the button styling: Active, Hover, and Pressed

Reply:
> - [ ] As requested by the previous reviewer and also a project requirement, Kindly implement the hover and pressed button state interactions to stick with the Figma design as possible. This provides visual feedback to users when they interact with the interface.☺
> 
> Figma design👇:
> 
> ![- pressed-figma](https://user-images.githubusercontent.com/40009354/235300233-775f21cc-145e-4d55-9b8a-25c44ab60367.PNG)
> 
> Your design👇:
> 
> ![- pressed](https://user-images.githubusercontent.com/40009354/235300388-76d69570-b41b-4c8d-99d4-3182159c2733.PNG)
>